### DISCUSSION TITLE
Run unreleased GoReleaser version in CI

### DISCUSSION THREAD

**Discussion by colinking** [goreleaser/goreleaser#4080] [Q&A] [answered]
We run GoReleaser from GitHub Actions and would like to run a version of GoReleaser that includes [this fix](https://github.com/goreleaser/goreleaser/issues/4011#issuecomment-1551951168). Is it possible to run GoReleaser in CI from a specific commit (or just the latest from the `main` branch)? If it's relevant, we're using the pro distribution.

**Comment by caarlos0** [comment, marked as answer]
Hey!

Specifically for the pro dist, yes, you can use the nightly builds: https://github.com/goreleaser/goreleaser-pro/releases/tag/nightly

FWIW: I just triggered a new build, should be done in ~30min.

**Reply by colinking** [reply]
Thanks! Are nightlies supported in the GitHub action? I tried `version: nightly`, `version: v1.19.0-pro-nightly`, and passing `--nightly` via `args`. The action appears to search https://goreleaser.com/static/releases.json which doesn't include the nightlies.

**Reply by caarlos0** [reply]
true, working on it : https://github.com/goreleaser/goreleaser-action/pull/419

**Reply by caarlos0** [reply]
action released!
